src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/ClaimIssuedEvent.java

```diff
@@ -35,8 +34,8 @@ public Claim getClaim() {
         return claim;
     }
 
-    public Optional<String> getPin() {
-        return Optional.ofNullable(pin);
+    public String getPin() {
```

Comment:
The getter can stay an `Optional` -- it's a method that returns a possibly blank value.